ClangPowerTools/ClangPowerTools/Commands/IgnoreCompileCommand.cs

```diff
@@ -73,8 +74,13 @@ public void RunIgnoreCompileCommand(int aId)
     {
       var task = Task.Run(() =>
       {
-        List<string> documentsToIgnore = ItemsCollector.GetDocumentsToIgnore();
-        AddIgnoreFilesToSettings(documentsToIgnore);
+        List<string> projectsToIgnore = ItemsCollector.GetProjectsToIgnore();
+        AddIgnoreProjectsToSettings(projectsToIgnore);
+        if (!projectsToIgnore.Any())
```

Comment:
For better code readability replace your ! with a standard boolean comparison

Reply:
Thank you for your suggestion. You say that "!projectsToIgnore.Any()" is less readable than "projectsToIgnore.Count() > 0".

I personally preffer .Any() rather than .Count() in this situation (performance reasons). Count() will check every element of the list, while Any() checks only the first one. 
https://www.dotnetjalps.com/2013/01/linq-performance-with-count-and-any.html

Reply:
.Any() returns a boolean value ( Any() == false). This is all you need to make an explicit comparison.

Reply:
Aaa, ok. I'll keep that in mind, thank you. Have a nice day!